src/components/BadgeEmoji/BadgeEmoji.js

```diff
@@ -5,11 +5,19 @@ import './badgeEmoji.scss'
 
 export default function Badge ({ emoji, expanded, className, common, border, onClick, name, id, responsibilities = [] }) {
   if (!emoji) return null
+
+  // TODO: why is this items?
```

Comment:
Basically so it could have accommodated responsibilities that had already been selected for their items and those that hadn't been...

Honestly an ongoing annoying pattern (we get the items thing because we use pagination) that we should straighten out once the urql stuff starts coming across